mul_table_func.py

```diff
@@ -0,0 +1,14 @@
+"""
+ Author: Elian
+ Purpose: Contains a multiply function.
+ Date: 11.03.2024
+"""
+
+
+def mul_table(num: int) -> list:
+    """
+    Return a list of the multiplied number.
+    :param num: The number to multiply.
+    :return: List of multiplied number.
+    """
+    return [num * i for i in range(1, num + 1)]
```

Comment:
For more generic code, we never use "magic numbers" like the number "1"
Think about other options to write this code without magic numbers

Reply:
This is one good option, think about one more